HW0/P6/P6.txt

```diff
@@ -1 +1,17 @@
 # Your answers here
+P6A:
+	The output of print Jobs has different order each time the script run.
+	After several times of running, I found the order of each sentence appearing is random.
+	I think the randomness happened because there is only one input flow for the I/O. 
+	Since the burnTime function is run on 4 processor simultaneous, the processors turn the data into I/O at the same time.
+	Then the I/O randomly pick one statement to print out.
```

Comment:
Once again, I agree with samighoche. Think of it this way: parallel jobs can process in parallel (lol redundancy), but a serial process must first occur to initialize the parallelization. My understanding is that each job theoretically has an expected order of completion, assuming perfect conditions across processors and equal communication distances/latency/etc; however, there are other processes, physical phenomena, and other noise that may slow down an individual processor, thereby leading to seemingly random outputs.